eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Deleg.hs

```diff
@@ -176,8 +171,8 @@ conwayDelegTransition = do
       Just deposit == fmap (UM.fromCompact . UM.rdDeposit) (UM.lookup stakeCred $ UM.RewDepUView dsUnified) ?! IncorrectDepositDELEG deposit
     checkStakeKeyNotAlreadyRegistered stakeCred dsUnified =
       UM.notMember stakeCred (UM.RewDepUView dsUnified) ?! StakeKeyAlreadyRegisteredDELEG stakeCred
```

Comment:
I just realized something. Shouldn't it be not yet registered? "Not already registered" does not sound right to me?

Let's remove this confusing naming. This will make it consistent with other predicate failures.
```suggestion
    checkStakeKeyNotRegistered stakeCred dsUnified =
      UM.notMember stakeCred (UM.RewDepUView dsUnified) ?! StakeKeyRegisteredDELEG stakeCredfied
```